ci-operator/step-registry/rosa/setup/rosa-setup-commands.sh

```diff
@@ -19,6 +19,45 @@ log(){
     echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") " "${*}"
 }
 
+override_rosa () {
+  # make a temp dir for rosa download
+  rosaDownloadTempDir=$(mktemp -d)
+  cd $rosaDownloadTempDir
+
+  # get the rosa downboad binary according to the version
+  wget https://github.com/openshift/rosa/releases/download/$ROSACLI_BUILD/rosa_Linux_x86_64.tar.gz
+  
+
+  tar -xvf rosa_Linux_x86_64.tar.gz
+  chmod +x ./rosa
+
+  # override the PATH 
+  echo "[CI] Overriding the rosa PATH in the image with new PATH $rosaDownloadTempDir/rosa"
+  export PATH=$rosaDownloadTempDir:$PATH
+  
+  # verify the rosa override
+  # I didn't use rosa version to verify because there is a known issue will cause nil pointer error for the command
+  # will update to rosa version to verify the overriden one once it is fixed.
+  echo "[CI] Verify current rosa is using the overrided PATH $rosaDownloadTempDir/rosa"
+  current_rosa_path=$(which rosa)
+  if [[ "$current_rosa_path" != "$rosaDownloadTempDir/rosa" ]];then
+    echo "[CI] rosa override failed. Current rosa is using $current_rosa_path not from $rosaDownloadTempDir/rosa"
+    exit 1
+  else
+    echo "[CI] rosa is overrided"
+  fi
+ 
+  rosa version
```

Comment:
Should be removed or comment out first?

Reply:
It had been fixed.

Reply:
Hmm... let me comment it in case any others using an old version